src/topology/algebra/open_subgroup.lean

```diff
@@ -55,30 +55,22 @@ variables {G : Type*} [group G] [topological_space G]
 variables {U V : open_subgroup G} {g : G}
 
 @[to_additive]
-instance has_coe_set : has_coe_t (open_subgroup G) (set G) := ⟨λ U, U.1⟩
-
-@[to_additive]
-instance : has_mem G (open_subgroup G) := ⟨λ g U, g ∈ (U : set G)⟩
+instance : set_like (open_subgroup G) G :=
```

Comment:
Can you make this a `subgroup_class` instead?

Reply:
`subgroup_class` is a `Prop` over `set_like`. I'll add an instance.